OpenRA.Mods.Common/UpdateRules/Rules/20190314/RenameSearchRadius.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	class RenameSearchRadius : UpdateRule
+	{
+		public override string Name { get { return "Rename SearchFromOrderRadius to SearchFromHarvesterRadius"; } }
+		public override string Description
+		{
+			get
+			{
+				return "Renamed 'SearchFromOrderRadius' to 'SearchFromHarvesterRadius'.";
+			}
+		}
+
+		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
+		{
+			var harvester = actorNode.LastChildMatching("Harvester");
```

Comment:
This can simplify all the way down to something like:
```csharp
public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
{
	foreach (var harvester in actorNode.ChildrenMatching("Harvester"))
		harvester.RenameChildrenMatching("SearchFromOrderRadius", "SearchFromHarvesterRadius");
	
	yield break;
}
```

This pattern has the advantage of being robust against multiple trait definitions.